src/controllers/bmdashboard/bmInventoryTypeController.js

```diff
@@ -174,6 +174,75 @@ function bmInventoryTypeController(InvType, MatType, ConsType, ReusType, ToolTyp
     }
   }
 
+  async function addToolType(req, res) {
+    const {
+      name,
+      description,
+      invoice,
+      purchaseRental,
+      fromDate,
+      toDate,
+      condition,
+      phoneNumber,
+      quantity,
+      currency,
+      unitPrice,
+      shippingFee,
+      taxes,
+      totalPriceWithShipping,
+      images,
+      link,
+      requestor: { requestorId },
+    } = req.body;
+
+    try {
```

Comment:
I believe we do not need try-catch block as all the code that is present within the try catch block is using `.then() and .catch()`.